cockroachcloud/use-managed-service-backups.md

```diff
@@ -286,6 +286,6 @@ Find the cluster backup you want to restore, and click **Restore**.
 
 Performing a restore will cause your cluster to be unavailable for the duration of the restore. All current data is deleted, and the cluster will be restored to the state it was in at the time of the backup. There are no automatic incremental backups, and no automatic database or table level backups.
 
-You can [manage your own backups](take-and-restore-customer-owned-backups.html), including incremental, database, and table level backups. To perform manual backups, you must configure either a [`userfile`](take-and-restore-customer-owned-backups.html) location or a [cloud storage location](take-and-restore-customer-owned-backups.html?filters=cloud), which requires [billing information](billing-management.html) for your organization even if you don't set a spend limit. 
+You can [manage your own backups](take-and-restore-customer-owned-backups.html), including incremental, database, and table level backups. To perform manual backups, you must configure either a [`userfile`](take-and-restore-customer-owned-backups.html) location or a [cloud storage location](take-and-restore-customer-owned-backups.html?filters=cloud), which requires [billing information](billing-management.html) for your organization even if you don't set a resource limit. 
```

Comment:
Not really about resource limits, but import/export should no longer require a paid plan.

Also, a general comment: it used to be that setting a spend limit is what triggered conversion to a paid plan. But that's not quite true with resource limits, since we now allow "unlimited". It's kind of weird to be talking about "setting a resource limit" as a kind of synonym for "have a paid plan" or "are a paid org".

Reply:
I think this was just outdated, should be updated to match the main docs now